Localization/Translations/Zones/EasternKingdoms.lua

```diff
@@ -327,14 +327,14 @@ local easternKingdomsLocales = {
         ["zhCN"] = "奥特兰克山脉",
     },
     ["Dalaran"] = {
-        ["ptBR"] = true,
+        ["ptBR"] = "Dalaran",
```

Comment:
`true` always means "take the key as translation". So if the key and the translation are the same it is better to leave it as `true`.

Reply:
That shipped sailed a while ago, every other translation put it in

Reply:
No reason to make it worse though.